Dokumentationen/architecture/architecture_notebook.adoc

```diff
@@ -144,6 +144,9 @@ a|  * Kompatibilität
 .. Objekte 
 .. Konfigurationen 
 
+. Weiterverwendung des bis jetzt genutzen Test-Frameworks und der bis jetzt existierenden (teilweise fehlerhaften) Testcases und damit einherkommende Fehlerbehebung
```

Comment:
```suggestion
. Weiterverwendung des bis jetzt genutzten Test-Frameworks und der bis jetzt existierenden (teilweise fehlerhaften) Testcases und damit einherkommende Fehlerbehebung
```

Reply:
Wir könnten hier nach `jetzt genutzten Test-Frameworks` in Klammern den Namen schreiben: `(RSpec)`